<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DisplayWidget" name="display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>580</width>
      <height>580</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести исходное изображение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться на шаг назад</string>
    </property>
   </widget>
   <widget class="QFrame" name="moveFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>250</width>
      <height>145</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="moveLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенос</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="moveDxLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
    <widget class="QLabel" name="moveDyLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveDxInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="moveDyInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="moveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rescaleFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>270</y>
      <width>250</width>
      <height>145</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rescaleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштабирование</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rescaleXLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Xц</string>
     </property>
    </widget>
    <widget class="QLabel" name="rescaleYLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Yц</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rescaleXInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rescaleYInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rescaleButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLabel" name="rescaleKyLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ky</string>
     </property>
    </widget>
    <widget class="QLabel" name="rescaleKxLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>30</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>kx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rescaleKyInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="rescaleKxInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rotateFrame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>250</width>
      <height>145</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rotateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Поворот</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rotateXLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Xц</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotateYLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>30</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Yц</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateXInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateYInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLabel" name="rotateAngleLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>35</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>угол</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rotateAngleInput">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayWidget</class>
   <extends>QGraphicsView</extends>
   <header>displaywidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
